.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
 	"search.exclude": {
 		"sites/svelte-5-preview/static/*": true
-	}
+	},
+	"typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
?? What's this? Telling VS Code not to use its built-in copy of TS?

Reply:
yeah, VSCode currently ships with 5.4, but you need 5.5 for `@import` to work. it's a good setting to have anyway IMHO